packages/core/src/sass/app/_common/components/account-switcher.scss

```diff
@@ -269,10 +303,13 @@
                     border: 1px solid var(--text-prominent);
                 }
             }
-        }
-        &--demo-text {
-            padding-top: 0.3em;
-            padding-bottom: 0.3em;
+            .acc-switcher__loginid-text {
```

Comment:
Can we group all the `.acc-switcher`s in this indent level and use `&` selectors for their element selection?